rsmtool/configuration_parser.py

```diff
@@ -689,6 +888,54 @@ def load_config_from_dict(self, config_dict):
                                  'been assigned. You cannot assign another'
                                  'dictionary.')
 
+        if configdir is None:
+            configdir = getcwd()
+
+        self._configdir = abspath(configdir)
+        logging.info("Reference directory configdir was set to {}".format(self._configdir))
```

Comment:
```suggestion
        logging.info("Configuration directory will be set to {}".format(self._configdir))
```

let's be consistent with the other logging message.